README.md

```diff
@@ -0,0 +1,21 @@
+# Installation #
+
+Sing up or sing in bitbucket.
```

Comment:
I wouldn't mention Bitbucket and generating keys in this case, otherwise we should mention that we have to grant access to the private repositories. I guess it's redundant. However, I would write somewhere in the README that the project depends on the code which is currently not available on GitHub. It's very important.